modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,400 @@
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Outbound Requests by Total Count::
+Number of failed outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-errors[]
+//-----------------------//
+//
+//-----------------------//
+// APP DETAIL JVM: JVM tab
+//tag::app-detail-jvm[]
+//TODO: clarify name (SATHISH)
+Old Gen GC Count::
+Number of times old generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: clarify name (SATHISH)
+Old Gen GC Duration::
+Amount of time taken for old generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: clarify name (SATHISH)
+Young Gen GC Count::
+Number of times young generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: clarify name (SATHISH)
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
+Amount of physical memory committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Usage::
+Amount of physical memory used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Thread Count::
+//TODO: threads that are available or used??? (SATHISH)
+Number of threads for simultaneous requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Committed Virtual Memory::
+Amount of memory (in MB) committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+CPU Utilization::
+CPU work as a percentage of total CPU time
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-jvm[]
+//------------------------------------------------------//
+
+//========================//
+//API DETAILS: ALL METRICS//
+//========================//
+//
+// API DETAILS: Header-level metrics
+//-------------------------
+//tag::api-detail-header[]
+
+//TODO: change to API (SATHISH)
+Type::
+API implementation type (*Api*)
+
+Status::
+Status code that indicates whether the governance policies and SLA tiers are functional (*Active*) or not
++
+For all codes, see information on xref:api-manager::latest-overview-concept.adoc#what-api-manager-looks-like[tracking registration status] in the API Manager UI documentation.
+
+Implementation URL:: 
+URL of deployed API integration
+
+//TODO: suggest Runtime Version conform on Runtime (App) vs. Runtime Version (API)
+//TODO:what If the runtime version is empty... (SEE BOOKS API IN STGX)
+//TODO: capitalize "version"?
+//(SATHISH)
+Runtime version::
+Mule Gateway or Flex Gateway runtime version used by the API implementation 
+
+//  shared header metric
```

Comment:
TODO: @dukesphere where is the entity-replicas tag defined? Is this missing from doc?